packages/core/src/codewhisperer/models/constants.ts

```diff
@@ -132,6 +132,8 @@ export const contextPreviewLen = 20
  */
 export const unsupportedLanguagesCacheTTL = 10 * 60 * 60 * 1000
 
+export const isAmazonQVisibleKey = 'AMAZON_Q_CHAT_VISIBLE'
```

Comment:
there is no reason for key names to be uppercaese. the ones below are legacy, that isn't the convention going forward. and it doesn't match most of the existing keys.
```suggestion
export const isAmazonQVisibleKey = 'amazonq.chat.visible'
```

Reply:
Will update in the next revision :)